src/index.js

```diff
@@ -21,6 +22,12 @@ exports.createRouter = (config = {}) => {
   router.use(bodyParser.json({}))
   router.post('/send', (req, res, next) => {
     const { body } = req
+    const token = body.token
+    if( token != null && token !== env('TOKEN')){
+      return res.status(404).json({
```

Comment:
If it returns forbidden, it should use the 403 code :+1:

Reply:
good catch!